Makefile

```diff
@@ -107,8 +107,8 @@ prepare: Dockerfile
 Dockerfile:
 	envsubst '$${BASE_NAMESPACE},$${BASE_IMAGE},$${BASE_TAG},$${CIRCLECI_USER}, \
 		$${YAMLLINT_VERSION},$${YQ_VERSION},$${BATS_VERSION},$${JUNIT_MERGE_VERSION},$${FLAKE8_VERSION}, \
-		$${TAP_XUNIT_VERSION},$${HADOLINT_VERSION},$${SHELLCHECK_VERSION},$${HELM2_VERSION},$${HELM3_VERSION}, \
-		$${TRIVY_VERSION},$${SHFMT_VERSION},$${GOOGLE_SDK_VERSION}$${NODE_VERSION}' \
+		$${OAUTHLIB_VERSION},$${TAP_XUNIT_VERSION},$${HADOLINT_VERSION},$${SHELLCHECK_VERSION},$${HELM2_VERSION}, \
```

Comment:
How are these env vars used? If they're not useful for any automation, it may be easier to just include the versions directly inside of the Dockerfile?

Reply:
These are defined in [config.default](https://github.com/greenpeace/planet4-circleci/blob/main/config.default#L21). I wouldn't call it an automation, but it helps to keep all variables in one place.

Reply:
The Dockerfile is also one place :grin:

Reply:
Yeah, but I don't think there is a way to define them in the header or in a way that more easier to parse when you need to update them. Also I think that some of these variables are used on the Makefile too.

Reply:
Well to update them you need to understand the code, which involves tracking the env var across 3 files. Especially this wall of all caps can cause some headache and we probably don't want it to grow too large :smile: https://github.com/greenpeace/planet4-circleci/blob/main/Makefile#L108-L111

Reply:
Is there a reason why `envsubst` doesn't read `config.default` as a file?

Reply:
I haven't tested that. It could work.